scanner/metadata/cuesheet/cuesheet.go

```diff
@@ -0,0 +1,675 @@
+package cuesheet
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/navidrome/navidrome/log"
+)
+
+const (
+	delims          = "\t\n\r "
+	eol             = "\n"
+	framesPerSecond = 75
```

Comment:
I'm not a Navidrome maintainer but thought I'd leave some comments...

I think it's worth adding a comment noting that `framesPerSecond = 75` is based on audio CD sectors and 44100 Hz sample rate (which I think is OK, since I've never seen cue sheets used for non-CD sourced files)

Reply:
Good catch. Will add comment.